<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotClass</class>
 <widget class="QWidget" name="ScreenshotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenshot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame
{
	background: #152034;
	border-radius: 8px;
}

QPushButton
{
	border-radius: 14px;
	border: 1px solid #01EEC3;
	font-size: 12px;
	font-family: 微软雅黑;
	color: #01EEC3;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_newScreen">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>75</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>api</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_screen">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>75</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>截图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>75</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_image">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0.6);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Screenshot.qrc"/>
 </resources>
 <connections/>
</ui>
